resources/geocoding/en/52.txt

```diff
@@ -1,819 +1,420 @@
-5255|Mexico City
-52222|Amozoc
-52222|Puebla
-52223|Magdalena Tetela
-52224|Atoyatempan
+# Copyright (C) 2011 The Libphonenumber Authors
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+# http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Generated from:
+# http://www.itu.int/oth/T020200008A/en
+# http://numeracion.ift.org.mx/numeracion.exe/w_edoymun
+#
+# For determine the city's geocoding:
+#    http://numeracion.ift.org.mx/numeracion.exe/info_asl?cld=<lada>
+#   This page shows "Ciudad con mayor cantidad de numeración asignada"
+
+
+52222|Heroica Puebla De Zaragoza
+52223|Tepeaca
+52224|Tecali De Herrera
+52225|Tlapacoyan
 52226|Altotonga
-52226|Jalacingo
 52227|Huejotzingo
-52228|Jalapa
-52232|La Vigueta
+52228|Xalapa-enriquez
+52229|Veracruz
+52231|Teziutlan
 52232|Martinez De La Torre
-52233|Cuetzalan
+52233|Zacapoaxtla
 52235|Misantla
-52235|Nautla
 52236|Ajalpan
-52236|Calipan
-52236|Cuicatlan
-52236|Huautla De Jimenez
+52237|San Gabriel Chilac
+52238|Tehuacan
 52241|Apizaco
-52241|Hueyotlipan
-52243|Atencingo
 52243|Izucar De Matamoros
 52244|Atlixco
-52244|La Venta Cacaloxuchil
-52245|Esperanza
+52245|Ciudad Serdan
+52246|Tlaxcala De Xicohtencatl
 52247|Huamantla
-52249|Acatzingo
-52249|Cañada Morelos
-52249|Los Reyes De Juarez
-52249|Palmar De Bravo
-52249|Palmarito Tochapan
-52271|Amatlan
-52271|Potrero
-52272|Maltrata
+52248|San Martin Texmelucan De Labastida
+52249|Tecamachalco
+52271|Cordoba
 52272|Orizaba
-52273|Chocaman
-52273|Huatusco
-52273|Ixhuatlan Del Cafe
-52273|Paso Del Macho
-52274|Acatlan De Perez Figueroa
+52273|Huatusco De Chicuellar
+52274|Tierra Blanca
```

Comment:
Ok, seems it would be 
52274|Vicente Camalote OAX/Tierra Blanca VER/Temascal OAX/Acatlan de Perez Figueroa, OAX

Which is even worse...